src/vs/workbench/browser/actions/windowActions.ts

```diff
@@ -343,7 +343,7 @@ export class NewWindowAction extends Action {
 	}
 
 	run(): Promise<void> {
-		return this.hostService.openWindow();
+		return this.hostService.openWindow({ remoteAuthority: null });
```

Comment:
Why do we need to introduce `null` to enforce local remote authority? If I leave out `remoteAuthority`, shouldn't that be sufficient?

Reply:
Oh I think I see it now, in `nativeHostService.doOpenWindow` you automagically add the `remoteAuthority` of the current window unless it is explicitly set to `null`.

Reply:
Yes, if undefined we use the one from the current window.